src/main/java/org/spongepowered/api/data/DataHolder.java

```diff
@@ -65,5 +65,5 @@
      * @throws InvalidDataException If the container is missing or has invalid
      *     data that this holder will refuse
      */
-    void setRawData(DataContainer container) throws InvalidDataException;
+    void setRawData(DataView container) throws InvalidDataException;
```

Comment:
Fixed it in a [branch of spongeforge](https://github.com/SpongePowered/SpongeForge/compare/feature/schematics)